portfolio/src/main/webapp/index.html

```diff
@@ -63,8 +63,22 @@ <h1>Omar Montiel's Portfolio</h1>
       <!-- Contact Section -->
       <div id="contact" class="section">
         <a href="resume.pdf" target="_blank">Resume</a>
-        <button onclick="getMessage()">Try Fetch</button> <!-- Fetch message from Data Servlet -->
-        <div id="message-container"></div> <!-- Display message from Data Servlet -->
+        <!-- Comment Input Form -->
+        <form action="/data" method="POST">
+          <p>Leave a comment:</p>
+          <textarea name="text-input">Say hi!</textarea>
+          <br/><br/>
+          <input type="submit" />
+        </form>
+        <br>
+        <!-- Comment Output Form -->
+        <form>
+            <label for="quantity">How many comments do you want displayed?</label>
+            <input type="number" id="quantity" name="quantity" min="0">
```

Comment:
idk exactly how this works, so maybe it already does this, but how does this handle invalid input? And what happens if the user does not input anything before clicking 'getComment'?

Reply:
If user inputs empty comment, the form still takes it, and it would be an empty string. Should I prevent this somehow? I don't know about invalid input, because I don't know what would be considered invalid input. Do you have examples of this?

Reply:
Bump.